### ISSUE TITLE
Improvments : a live viewer of notifications ?

### ISSUE THREAD

**Issue by rigobert9** [AlphaTechnolog/notification-monitor#2]
I have no idea what is the scope of the project, but since it is what I was searching for, I wondered if there was a plan of adding a window that can kind of "tail" the notifications and manage them.

**Comment by AlphaTechnolog** [comment]
Yeah, I did this, because I have an idea for my dotfiles, in my dotfiles I want to put a notifications manager, and this will be the backend.

Later I will create a graphical application that do the same that the client :3 (that only works for now in terminal based applications)

**Comment by AlphaTechnolog** [comment]
But yeah, a graphical application will be a good improvement
